Docs/reference/content/reference/driver/admin.md

```diff
@@ -189,6 +189,45 @@ collection.Indexes.CreateOne("{ x : 1 }", options);
 await collection.Indexes.CreateOneAsync("{ x : 1 }", options);
 ```
 
+Also, there is a generic [`CreateIndexOptions<TDocument>`]({{< apiref "T_MongoDB_Driver_CreateIndexOptions_1" >}}) which extends the non-generic version and provides a number of additional options available when creating index. For example, to create a `wildcard projection` index on "name" and "description.text", do the following:
+
+Given the following classes:
+
+```csharp
+class Description
+{
+	[BsonElement("text")]
+	public string Text { get; set; }
+}
+
+class Widget
+{
+	[BsonElement("name")]
+	public int Name { get; set; }
```

Comment:
Indent 4 spaces instead 1 tab. Check all similar lines.

Reply:
Ok, just to mention, as I see we use both ways.
Done

Reply:
We should never use tabs. If there are places where we do it is a mistake.